.env

```diff
@@ -1,13 +1,19 @@
-// Answer Key Directions
-// The Instrumenting with Node.js Using OpenTelemetry course on Honeycomb Academy gives you instructions on making code changes to this respository to implement instrumentation to the backend-for-frontend service. 
-// The code changes are commented out. 
-// Each code change includes a reference back to the activity in the course that contains instructions for the code change.
+# Answer Key Directions
+# The Instrumenting with Node.js Using OpenTelemetry course on Honeycomb Academy gives you instructions on making code changes to this repository to implement instrumentation to the backend-for-frontend service.
+# The code changes are commented out.
+# Each code change includes a reference back to the activity in the course that contains instructions for the code change.
 
-// Step 1 of Automatic Instrumentation with OpenTelemtery for a Node.js service. Add your API key
+# Step 1 of Automatic Instrumentation with OpenTelemtery for a Node.js service. Add your API key
 HONEYCOMB_API_KEY="your-api-key"
 
 # you could change this to your own bucket of images. We accept no responsibility for the outcome.
 BUCKET_NAME="random-pictures"
 
 OTEL_EXPORTER_OTLP_ENDPOINT="https://api.honeycomb.io:443/"
 OTEL_EXPORTER_OTLP_HEADERS="x-honeycomb-team=${HONEYCOMB_API_KEY}"
+
+# Emit both old and new (stable) http attributes from semantic conventions
+# for example, http.status_code and http.response.status_code
+# When ready to use all new attributes, set this to "http"
+# see https://opentelemetry.io/docs/specs/semconv/non-normative/http-migration/
+OTEL_SEMCONV_STABILITY_OPT_IN="http/dup"
```

Comment:
Add this line to ensure you get both old and new HTTP attributes from semantic conventions.